pyclient/pydeephaven/__init__.py

```diff
@@ -25,10 +25,10 @@
 from .table import Table
 from .session import Session
 from .dherror import DHError
-from ._combo_aggs import ComboAggregation
+from .combo_agg import ComboAggregation
 from .constants import SortDirection, MatchRule
-from ._table_interface import TableInterface
+from .table_interface import TableInterface
 from .query import Query
 
 __version__ = "0.5.0"
-__all__ = ["Session", "Table", "Query", "TableInterface", "ComboAggregation", "DHError", "SortDirection", "MatchRule"]
+# __all__ = ["Session", "Query", "TableInterface", "ComboAggregation", "MatchRule", "SortDirection"]
```

Comment:
This is commented out.  I assume that we either want something or nothing.  A commented out line is a weird middle that should be avoided.